test/addons_test.go

```diff
@@ -260,6 +315,35 @@ func cleanupNodeVolumes(numberVolumes int, nodePrefix string, node *v1alpha4.Nod
 }
 
 func testgroup(t *testing.T, groupname string, version string, jobs ...clusterTestJob) error {
+	var err error
+	t.Logf("testing deployment group %s", groupname)
+
+	t.Logf("=== Running INSTALL job")
+
+	// TODO these happen sequentially, we should do so in parallel when we have enough confidence
+	err = testGroupDeployment(t, groupname, version, jobs)
+	if err != nil {
+		return err
+	}
+
+	doUpgrade, addonDeployments, err := checkIfUpgradeIsNeeded(t, groupname)
```

Comment:
prefer calling this `addons`, as it's making me think of the old `addonDeployments`, but it's a nit

Reply:
hmmm, yeah i see what you mean, we can still make that change.